packages/validation/src/validators/LionWebReferenceValidator.ts

```diff
@@ -191,10 +185,9 @@ export class LionWebReferenceValidator {
                 if (childNode !== undefined) {
                     if (childNode.parent !== node.id) {
                         // TODO Check that this is already tested from the child in vaidateExistsAsChild().
-                        // this.validationResult.error(`PP Parent of ${context}.${child.containment.key}[${index}] with id "${childId}" is "${childNode.parent}", but should be "${node.id}"`);
+                        //      this.validationResult.error(`PP Parent of ${context}.${child.containment.key}[${index}] with id "${childId}" is "${childNode.parent}", but should be "${node.id}"`);
```

Comment:
This comment could be removed as well, right?

Reply:
Removed